Directory.Build.props

```diff
@@ -9,18 +9,22 @@
     <EmbedUntrackedSources>true</EmbedUntrackedSources>
     <GenerateDocumentationFile>true</GenerateDocumentationFile>
     <HighEntropyVA>true</HighEntropyVA>
-    <LangVersion>8.0</LangVersion>
+    <LangVersion>9.0</LangVersion>
```

Comment:
We'll use C# 9 explicitly, as we multi-target and by default the language is determined by the target framework